cmd/workspace/apps/run-local.go

```diff
@@ -0,0 +1,205 @@
+package apps
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/appproxy"
+	"github.com/databricks/cli/libs/apps"
+	"github.com/databricks/cli/libs/auth"
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/spf13/cobra"
+)
+
+var (
+	port               int
+	debug              bool
+	prepareEnvironment bool
+	entryPoint         string
+	customEnv          []string
+)
```

Comment:
Using globals for these options means their values will race for in-process testing.

Other commands use stack variables to scope them to the instance of the command.